pkg/audit/authreq_test.go

```diff
@@ -6,9 +6,23 @@ import (
 	"testing"
 
 	"github.com/openshift/oauth-server/pkg/audit"
+	kaudit "k8s.io/apiserver/pkg/audit"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 )
 
+func mustRequest(t *testing.T) *http.Request {
```

Comment:
either document this or name it better

Reply:
Oh, as this func doesn't have any input variables... it is either a const or has side-effects... :thinking:

Reply:
Name adjsuted